samples/XCT.Sample/ViewModels/Views/BadgeViewViewModel.cs

```diff
@@ -6,6 +6,7 @@ namespace Xamarin.CommunityToolkit.Sample.ViewModels.Views
 	public class BadgeViewViewModel : BaseViewModel
 	{
 		int counter;
+		float cornerRadius;
```

Comment:
I suggest using `double` since it's the type used by Xamarin.Forms as you can see [here](https://docs.microsoft.com/en-us/dotnet/api/xamarin.forms.cornerradius.-ctor?view=xamarin-forms)

Reply:
![image](https://user-images.githubusercontent.com/33021114/150180533-d04f177a-8a14-40ae-8311-3fd6d853a628.png)
Frame Corner radius is float

Reply:
Hm... We missed that on the first Code Review, we can change it on the `Badge` control and release this one on v2.0 as a breaking change... Or we can leave at it's and change it on MCT. Both work for me, what do you think?

Reply:
cc: @jfversluis @jsuarezruiz